### DISCUSSION TITLE
Şehir şifre sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#304] [Zeka Sorusu] [answered]
konya=20
istanbul=31
muş=10
bilecik=?
bitlis=22

**Comment by ArvenAA** [comment, marked as answer]
İlk sayılar şehir isminde kaç tane sesli harf olduğunu ifade ederken, ikinci sayılar ise şehir isminin içinde kaç tane ''i'' harfi olduğunu ifade eder. Yani ''bilecik''in sayısı 32'ye eşittir, üç tane sesli harf (i,e,i) ve iki tane ''i'' harfi.
